client/views/my-streets/components/my-streets.js

```diff
@@ -25,10 +34,13 @@ export default class MyStreets extends React.Component {
     constructor(props) {
         super(props);
         this.state = {
+            isMember: false,
             postfeed: [],
             currentWriter: '',
-            members: ['Idan', 'Michael', 'Daniel'],
+            members: [],
         };
+
+        this.onJoinClick = this.onJoinClick.bind(this);
```

Comment:
Again, this shouldn't be necessary as `onJoinClick` is implemented using an arrow function.